src/Converter.php

```diff
@@ -325,6 +323,8 @@ private static function convertLearnosityItem(array $itemJson)
         $item = $itemMapper->parse($itemJson);
         $questions = [];
         foreach ($questionsJson as $question) {
+            $question['reference'] = $question['response_id'];
```

Comment:
What's this change needed for? Is this also related to the missing `reference` data that you mentioned here?
https://github.com/Learnosity/learnosity-qti/pull/34#discussion_r295651473

Reply:
Yes , changed now.